Components Scaffolding

chethanagopinath opened the pull request:
Looks like this right now, based on the data

![Screen Shot 2022-06-27 at 4 05 03 PM](https://user-images.githubusercontent.com/10332729/176035398-f413f339-6252-4a27-8227-3f936603a5af.png)


Need to clean up a lot of the defaults

jraigosaOC commented on public/index.html:
```diff
@@ -14,7 +14,7 @@
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
-    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <!-- <link rel="manifest" href="%PUBLIC_URL%/manifest.json" /> -->
```
maybe just 🔪 ?

jraigosaOC replied:
since we're removing the file

chethanagopinath replied:
👍🏼

jraigosaOC commented on src/Prompt.js:
```diff
@@ -0,0 +1,13 @@
+    );
+}
+
+export default Prompt
```
think the file structure is good for now.  I'm thinking we should a components directory under src and this should go in there. just to keep it a little tidier

jraigosaOC commented on src/App.js:
```diff
@@ -1,24 +1,60 @@
+      }
+    ]
+    }
+  ]
```
good w/ this for now.  think we might want to move this into a separate json data file.  there actually might a utility for turning flow charts into structured json.  will take a look.

jraigosaOC reviewed (commented):
A few comments.  I think the only thing gating for me is adding a components folder w/ prompt.js in it.

jraigosaOC commented on src/App.js:
```diff
@@ -1,24 +1,14 @@
-        </a>
-      </header>
-    </div>
+    questions.map((questionPrompt) => {
```
👍

jraigosaOC reviewed (approved):
🎸

chethanagopinath commented:
made some changes + addressed comments 

question - do we need the `robots.txt` as we aren't in scope of web scraping or anything for now 🙈 also do we need `reportWebVitals.js` and `setupTests.js`?  

feels like `package.json` also looks intense, but maybe we can clean that up later

may be good for a second glance (oooo :thank_you-8094: :juan-sunshine: !)

kelseyhuse30 reviewed (approved):
![](https://media.giphy.com/media/MbAKbcBhrgJoxNM6GT/giphy.gif)

chethanagopinath commented:
test

meganweijiang commented:
> made some changes + addressed comments
> 
> question - do we need the robots.txt as we aren't in scope of web scraping or anything for now 🙈 also do we need reportWebVitals.js and setupTests.js?
> 
> feels like package.json also looks intense, but maybe we can clean that up later
> 
> may be good for a second glance (oooo :thank_you-8094: folks!)

I think we can worry about clean up later; the most important thing is getting the app working 😆

meganweijiang reviewed (approved):
❤️
